RealmSwift/RealmConfiguration.swift

```diff
@@ -223,7 +223,11 @@ extension Realm {
             configuration.schemaVersion = self.schemaVersion
             configuration.migrationBlock = self.migrationBlock.map { accessorMigrationBlock($0) }
             configuration.deleteRealmIfMigrationNeeded = self.deleteRealmIfMigrationNeeded
-            configuration.shouldCompactOnLaunch = self.shouldCompactOnLaunch.map(ObjectiveCSupport.convert)
+            if let shouldCompactOnLaunch = self.shouldCompactOnLaunch {
```

Comment:
Please rebase this PR against `master`. You shouldn't need to touch `RealmConfiguration.swift` in this patch.